anitab-org/bridge-in-tech-web
# Feat view user programs list and program

### Description
Create View programs list and program profile functionalities

Fixes #75 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)


### How Has This Been Tested?
WIP
- login as a user who has an organization that already has a program (you can create this using BIT backend on Swagger UI as the frontend Create/Update program have not yet completed at this stage)
- go to `My Space` > `My Organization` to view the organization profile
- go to the bottom of the organization profile and click the `Go to Portfolio` button. See the programs list
<img width="1642" alt="Screen Shot 2020-08-24 at 1 22 47 am" src="https://user-images.githubusercontent.com/29667122/90982728-6895ba00-e5ac-11ea-8110-32484feaf714.png">

- select a program from the list and seee the program profile page
<img width="1610" alt="Screen Shot 2020-08-24 at 1 23 12 am" src="https://user-images.githubusercontent.com/29667122/90982744-91b64a80-e5ac-11ea-8f77-a1876f80294c.png">


### Checklist:

**Code/Quality Assurance Only**


**IMPORTANT** 
To test this PR, you need to run the bit backend server from the branch on [PR#116](https://github.com/anitab-org/bridge-in-tech-backend/pull/116) **ONLY** as this PR has pages with date and timezone fields that are already link to the refactored date fields on backend. (PS: don't forget, although not mentioned, you are **ALWAYS** expected to also run MS backend server using **ONLY** [ms-backend-server](https://github.com/mtreacy002/mentorship-backend/tree/ms-backend-server) with any of BIT Frontend PRs)
